pkgs/tools/virtualization/aws-iot-securetunneling-localproxy/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake, boost, protobuf, openssl, catch2 }:
+
+stdenv.mkDerivation rec {
+  pname = "aws-iot-securetunneling-localproxy";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner  = "aws-samples";
+    repo   = pname;
+    rev    = "v${version}";
```

```suggestion
    owner = "aws-samples";
    repo = pname;
    rev = "v${version}";
```
our formatting tools do not suppor this.